octodns/provider/route53.py

```diff
@@ -83,46 +150,235 @@ def __repr__(self):
         return '_Route53Record<{} {} {} {}>'.format(self.fqdn, self._type,
                                                     self.ttl, self.values)
 
+    def _value_convert_value(self, value, record):
+        return value
```

Comment:
All of these converts were broken out of the `_values_for_*` functions so they could be used directly when converting each `dynamic` value for a record based on its type.